administration/templates/administration/overview.html

```diff
@@ -36,19 +36,26 @@
     </script>
 {% endblock %}
 
-{% block content %}
-<div class="uu-inner-container">
-    <div class="col-12">
-        <div class="float-right">
-            <a class="button button-colored" href="{% url 'administration:ldap' %}">
+{% block pre-messages-content %}
+    <div class="uu-hero justify-content-between align-items-center">
+        <h1 class="h2">
+            {% trans 'experiments:home:header' %}
+        </h1>
+
+        <div class="btn-group" role="group">
+            <a class="btn btn-secondary btn-sm" href="{% url 'administration:ldap' %}">
                 {% trans 'administration:home:server_config' %}
             </a>
-        </div>
-        <div class="float-right mr-2">
-            <a class="button button-colored" href="{% url 'admin:index' %}">
+            <a class="btn btn-secondary btn-sm" href="{% url 'admin:index' %}">
                 {% trans 'administration:home:django_admin' %}
             </a>
         </div>
```

Comment:
This works and does not require a change, but it might be nice to know:
Adding the ``btn-group-sm`` class to ``btn-group`` will apply ``btn-sm`` to all child buttons. That way you don't have to apply it to the buttons individually;